molecule/default/tasks/json_patch.yml

```diff
@@ -0,0 +1,170 @@
+- vars:
+    namespace: json-patch
+    pod: json-patch
+    deployment: json-patch
+
+  block:
+    - name: Ensure namespace exists
+      kubernetes.core.k8s:
+        kind: namespace
+        name: "{{ namespace }}"
+
+    - name: Create a simple pod
+      kubernetes.core.k8s:
+        definition:
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            namespace: "{{ namespace }}"
+            name: "{{ pod }}"
+            labels:
+              label1: foo
+          spec:
+            containers:
+              - image: busybox:musl
+                name: busybox
+                command:
+                  - sh
+                  - -c
+                  - while true; do echo $(date); sleep 10; done
+        wait: yes
+
+    - name: Add a label and replace the image in checkmode
```

Comment:
Could we add a case that tests the `remove` action and verifies it's idempotent?

Reply:
Hmm, I'm not sure if we can test idempotency for remove ops. The path won't exist the second time around and kubernetes will return an error.